src/ProjectTemplates/GeneratedContent.targets

```
@@ -15,22 +15,24 @@
   <Target Name="ComputeGeneratedContentProperties">
     <PropertyGroup>
       <!-- Define optional pinned versions of certain dependencies. -->
-      <TemplatePinnedRepoPackagesVersion>9.3.0-preview.1.25161.3</TemplatePinnedRepoPackagesVersion>
-      <TemplatePinnedMicrosoftEntityFrameworkCoreSqliteVersion>9.0.3</TemplatePinnedMicrosoftEntityFrameworkCoreSqliteVersion>
+      <TemplatePinnedRepoPackagesVersion>9.4.0</TemplatePinnedRepoPackagesVersion>
+      <TemplatePinnedRepoAIPackagesVersion>9.4.0-preview.1.25207.5</TemplatePinnedRepoAIPackagesVersion>
```

```suggestion
      <TemplatePinnedMicrosoftExtensionsAIPackagesVersion>9.4.0-preview.1.25207.5</TemplatePinnedMicrosoftExtensionsAIPackagesVersion>
```
Better name aligned with MS.EFCore.Sqlite one?